R/linter_tags.R

```diff
@@ -3,6 +3,8 @@
 #' Obtain a tagged list of all Linters available in a package.
 #'
 #' @param packages A character vector of packages to search for linters.
+#' @param tags Optional character vector of tags to search. Only linters with at least one matching tag will be
+#' returned. If `tags` is `NULL`, all linters will be returned.
```

Comment:
Should we export an `available_tags()` too to help downstreams align to our terminology? Just a simple wrapper like

```
available_tags <- function(packages = "lintr") {
  unique(unlist(available_linters(packages = packages)[["tags"]]))
}
```

But saves them the effort of knowing how to do this & raises the prominence of it.

Feel free to mark as a follow-up.

Reply:
Sounds good to me.